src/content-script/actions/party.ts

```diff
@@ -1,16 +1,29 @@
-import { PartyActions } from '@root/lib/constants';
-// import { StoreState } from '@popup/store';
+import { PartyActions, SocketEvents } from '@root/lib/constants';
+import { debug } from '@root/lib/utils/debug';
 
-export const setRoomId = (roomId: string) => {
+import { PartyState } from '@contentScript/reducers/party';
+import socket from '@contentScript/socket';
+
+import { Dispatch } from 'redux';
+
+import camelCase from 'camelcase-keys';
+
+export const setParty = (details: PartyState) => {
   return {
-    type: PartyActions.SET_ROOM_ID,
-    roomId,
+    type: PartyActions.SET_PARTY,
+    ...details,
   };
 };
 
-export const setJoinUrl = (joinUrl: string | null) => {
-  return {
-    type: PartyActions.SET_JOIN_URL,
-    joinUrl,
+export const joinParty = ({ hash, isHost }: any) => {
```

Comment:
so is this changing the join_URL to be a hash for the server side? or client side as well?

Reply:
> so is this changing the join_URL to be a hash for the server side? or client side as well?

The hash and join_url are generated and stored on the server.